plugins/modules/dynamodb_table_info.py

```diff
@@ -0,0 +1,269 @@
+            returned: if any, on the table
+            type: complex
+            contains:
+                stream_enabled:
+                    description: Indicates whether DynamoDB Streams is enabled (true) or disabled (false) on the table.
+                    type: bool
+                    returned: always
+                    sample: true
+                stream_view_type:
+                    description: When an item in the table is modified, stream_view_type determines what information is written to the stream for this table.
+                    type: str
+                    returned: always
+                    sample: KEYS_ONLY
+        latest_stream_label:
+            description: A timestamp, in ISO 8601 format, for this stream.
+            type: str
+            returned: if any on the table
+        latest_stream_arn:
+            description: The Amazon Resource Name (ARN) that uniquely identifies the latest stream for this table.
+            returned: if any on the table
+            type: str
+        global_table_version:
+            description: Represents the version of global tables in use, if the table is replicated across AWS Regions.
+            type: str
+            returned: if the table is replicated
+        replicas:
+            description: Represents replicas of the table.
+            type: list
+            elements: dict
+            returned: if any on the table
+        source_backup_arn:
+            description: The Amazon Resource Name (ARN) of the backup from which the table was restored.
+            type: str
+            returned: if any, on the table
+        source_table_arn:
+            description: The ARN of the source table of the backup that is being restored.
+            type: str
+            returned: if any, on the table
+        restore_date_time:
+            description: Point in time or source backup time.
+            type: str
+            returned: if any, on table
+        restore_in_progress:
+            description: Indicates if a restore is in progress or not.
+            type: bool
+            returned: if any, on table
+        table_class:
+            description: The table class of the specified table.
+            type: str
+            returned: if any on the table
+            sample: STANDARD_INFREQUENT_ACCESS
+        deletion_protection_enabled:
+            description: Indicates whether deletion protection is enabled (true) or disabled (false) on the table.
+            type: bool
+            returned: always
+            sample: true
+        provisioned_throughput:
+            description: The provisioned throughput settings for the table.
+            type: dict
+            returned: always
+            sample: '{"number_of_decreases_today": 0, "read_capacity_units": 1, "write_capacity_units": 1}'
+        tags:
+            description: A dict of tags associated with the DynamoDB table.
+            returned: always
+            type: dict
```

Comment:
It seems like there are some values boto3 could return that aren't listed here, like `SSEDescription` and `ArchivalSummary`.

Reply:
@hakbailey I have added the missing elements into the output